oem/dell/dell.go

```diff
@@ -0,0 +1,52 @@
+package dell
```

Comment:
Can you add the license header to these files?

Reply:
done